lockfile.go

```diff
@@ -13,7 +13,11 @@ import (
 // identifier of the last party that made changes to whatever's being protected
 // by the lock.
 type Locker interface {
-	sync.Locker
+	Lock()
+
+	Unlock()
+
+	//	RLock()
```

Comment:
Why the comment?

Reply:
Testing. We need that when the PR progresses. The interface needs to change to expose RLock() but not all implementations of the interface yet support it.